fitbit-react/src/index.js

```diff
@@ -18,9 +19,11 @@ ReactDOM.render(
       <UserProvider>
         <ProductProvider>
           <FilterProvider>
-            <CartProvider>
-              <App />
-            </CartProvider>
+            <WishListProvider>
+              <CartProvider>
+                <App />
+              </CartProvider>
+            </WishListProvider>
           </FilterProvider>
         </ProductProvider>
       </UserProvider>
```

Comment:
Because there are so many contexts you can combine them into one

Reply:
Will try that